ecosystem/sep-0009.md

```diff
@@ -87,6 +87,14 @@ Name | Type | Description
 `organization.website` | string | Organization website
 `organization.email` | string | Organization contact email
 `organization.phone` | string | Organization contact phone
+`organization.individual_beneficial_owners.{index}.{SEP-0009 person field}` | array | Organization individual beneficial owners and control person
```

Comment:
Because the term `person` is used instead of `individual` in this SEP I would suggest using that term here instead too so that we maintain consistent ubiquitous language for the concept of a single person.